spec/features/content_data/content_data_non_beta_partner_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+RSpec.feature "No link to content data app if user is not a content data beta partner" do
+  scenario do
+    given_i_am_not_a_content_data_beta_partner
+    and_there_is_a_published_edition
+    when_i_visit_the_summary_page
+    then_i_dont_see_a_link_to_the_content_data_page_for_the_document
+  end
+
+  def given_i_am_not_a_content_data_beta_partner
+    login_as(create(:user, organisation_slug: "not-a-beta-partner"))
+  end
+
+  def and_there_is_a_published_edition
+    @edition = create(:edition, :published)
+  end
+
+  def when_i_visit_the_summary_page
+    visit document_path(@edition.document)
+  end
+
+  def then_i_dont_see_a_link_to_the_content_data_page_for_the_document
+    expect(page).not_to have_link("View data about this page")
+  end
+end
```

Comment:
Since the logic is encapsulated in the ContentDataUrl class I think we probably don't need this test and the recently published one (sorry - I hope they didn't take long to write)

Reply:
I've deleted both of these files.